NMS-12999: Move Provisiond adapter docs to Reference (#3693)
OpenNMS/opennms: docs/modules/reference/pages/provisioning/adapters/ddns.adoc

[[adapter-ddns]]
= Dynamic DNS

The DDNS adapter uses the dynamic DNS protocol to send updated A/AAAA DNS records to your DNS system as nodes are provisioned into {page-component-title}.

> For every IP address found, there must be a valid forward DNS zone created on your DNS server.

Author: ```suggestion
For every IP address found, a corresponding forward DNS zone must exist on your DNS server.
```
